quickff/program.py

```diff
@@ -264,8 +264,23 @@ def do_pt_generate(self):
             #configure
             self.reset_system()
             only = self.settings.only_traj
-            if only is None or only=='PT_ALL' or only=='pt_all':
+            dont_terms = self.settings.dont_terms
```

Comment:
To show clear analogy with settings.only_traj, I would suggest to name it settings.dont_traj